src/Hide/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import mediaQueries from "../utils/mediaQuery";
+import { DEVICES } from "../utils/mediaQuery/consts";
+
+import type { Props } from "./index";
+
+const sortedBreakpoints = Object.keys(DEVICES).sort((a, b) => DEVICES[a] - DEVICES[b]);
+
+const StyledHide = styled.div`
+  ${({ on, inline }) =>
+    sortedBreakpoints.map(
+      viewport =>
+        viewport in mediaQueries
+          ? css`
+              ${mediaQueries[viewport](css`
+                display: ${on.indexOf(viewport) !== -1
```

Comment:
checking if is element in array can be done by includes. Something like: on.includes(viewport).

Reply:
Array.includes is not supported in IEs 😞

Reply:
@wEprous another options is: ```if (~[1, 2, 3].indexOf(1))```. In this case ```indexOf(viewport) !== -1``` is the best decision since includes is not supported by IE11 without babel-polyfill, just share practice :)